erpnext/accounts/doctype/account/account_tree.js

```diff
@@ -56,7 +56,7 @@ frappe.treeview_settings["Account"] = {
 	toolbar: [
 		{
 			condition: function(node) {
-				return !node.root && frappe.boot.user.can_read.indexOf("GL Entry") !== -1
```

Comment:
`can_read` is false because of "User cannot search", which was enabled for gl entry for dynamic link validation. We can remove the permission check, the all the balances will be open to all. We need to check permissions in some other way.